src/module/Entity/src/Entity/Entity/Entity.php

```diff
@@ -253,6 +253,11 @@ public function isUnrevised()
         return $this->countUnrevised() > 0;
     }
 
+    public function isDead()
+    {
+        return !$this->getHead();
```

Comment:
That's weird, because if no head exists (no revision), there should also not be a current revision